### DISCUSSION TITLE
How do I make changes to config file(s)

### DISCUSSION THREAD

**Discussion by jmc1712** [lllyasviel/Fooocus#3484] [Q&A] [answered]
I would like to make some changes like only having 1 image vs 2 being created and I also want to add a few Aspect Ratios to what is available, can I do this?

Thanks

JC

**Comment by mashb1t** [comment]
Yes, of course you can do this.

**Reply by jmc1712** [reply]
I tried before the last update and every time I started Fooocus it went back to default and I just gave up. It seems the Aspect is now in a file called flag, correct?

**Reply by mashb1t** [reply]
@jmc1712 Fooocus loads the defaults from the modules/flags.py file, but you should adjust the config.txt instead of adjusting the code.
Please read the output of the terminal after adjusting the config to see if there are any errors or you do use invalid syntax.

**Reply by jmc1712** [reply]
I will make changes to the config_modification_tutorial and save it as config.txt since it has more info than the config I use which is just redirecting where the Models directory is as I have it on C but run software from D. Thanks.

**Reply by Yogesh-DevHub** [reply, marked as answer]
Here a 2min video on how to add custom aspect Ratios e.g.: 500*500, might be helpful.

https://youtu.be/ZLSJ44bvyxs?t=336&si=JMkeZwx4EHBqm6AM

**Reply by jmc1712** [reply]
Thanks.
